docs/UpgradeGuide.md

```diff
@@ -222,11 +323,11 @@ Datadog::Tracing.correlation
 
 Use of `active_root_span` has been removed.
 
-### Distributed tracing
+<h3 id="1.0-trace-api-distributed">Distributed tracing</h3>
 
 Previously, distributed tracing required building new `Datadog::Context` objects, then replacing the context within the tracer.
 
-Instead, users utilize `TraceDigest` objects derived from a trace. This object represents the state of a trace. It can be used to propagate a trace across execution boundaries (processes, threads), or to continue a trace locally.
+Instead, users utilize `TraceDigest` objects derived from a trace. This object represents the state of a trace. It can be used to propagate a trace across execution boundaries (processes, threads) or to continue a trace locally.
```

Comment:
I prefer shorter statements to conjunction here.

Reply:
Maybe then:

```suggestion
In 1.0, users need to use `TraceDigest` objects derived from a trace. `TraceDigest` represents the state of a trace. It can be used to propagate a trace across execution boundaries (processes, threads) or to continue a trace locally.
```